python_modules/dagster/dagster/_core/definitions/declarative_scheduling/asset_condition.py

```diff
@@ -419,13 +420,21 @@ def materialized() -> "AssetCondition":
         return MaterializedSchedulingCondition()
 
     @staticmethod
-    def in_latest_time_window() -> "AssetCondition":
+    def in_latest_time_window(
+        lookback_timedelta: Optional[datetime.timedelta] = None,
+    ) -> "AssetCondition":
         """Returns an AssetCondition that is true for an asset partition when it is within the latest
         time window.
+
+        Args:
+            lookback_timedelta (Optional, datetime.timedelta): If provided, the condition will
+                return all partitions within the provided delta of the end of the latest time window.
```

Comment:
Explicitly document edge conditions (Daily partition with 24 hour lookback) so that there is no doubt